install_config/master_node_configuration.adoc

```diff
@@ -1422,6 +1410,33 @@ start on a machine that does not have docker started.
 
 |===
 
+[[node-config-volume-config]]
+=== Volume Configuration
+
+You can use the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/storage_administration_guide/ch-xfs[XFS quota subsystem] 
+to limit the number of `emptyDir` volumes and volumes based on an `emptyDir` volume, such as secrets and configmaps, on each node.
+
+Enable a quota on local storage for each unique xref:../architecture/additional_concepts/authorization.adoc#security-context-constraints[FSGroup ID] 
+using the *_/etc/origin/node/volume-config.yaml_* file.
```

Comment:
Probably add this here:

NOTE: The `volume-config.yaml` file should not be edited directly as it is created from the `node-config-compute` configmap in the `openshift-node` namespace.  Creating or editing `volume-config.yaml` should be done via the configmap and not directly on the node.

Reply:
@sjenning What does the code in the config map configure  perFSGroup look like?

Reply:
An example:
```
apiVersion: kubelet.config.openshift.io/v1
kind: VolumeConfig
localQuota:
  perFSGroup: 1Gi
```